plugin/pkg/scheduler/scheduler.go

```diff
@@ -264,15 +263,16 @@ func (sched *Scheduler) scheduleOne() {
 
 	// Tell the cache to assume that a pod now is running on a given node, even though it hasn't been bound yet.
 	// This allows us to keep scheduling without waiting on binding to occur.
-	err = sched.assume(pod, suggestedHost)
+	assumedPod := *pod
```

Comment:
Why you make a copy here? seems you moved the copy from inside the assume func here.